resources/definitions/zonestar_z9m4_printer.def.json

```diff
@@ -0,0 +1,43 @@
+{
+    "name": "ZONESTAR Z9M4",
+    "version": 2,
+    "inherits": "fdmprinter",
+    "metadata": {
+        "visible": true,
+        "author": "ZONESTAR",
+        "manufacturer": "ZONESTAR",
+        "file_formats": "text/x-gcode",
+        "platform_offset": [ 0, 0, 0],
+        "machine_extruder_trains":
+        {
+            "0": "zonestar_printer_extruder_0",
+            "1": "zonestar_printer_extruder_1",
+            "2": "zonestar_printer_extruder_2",
+            "3": "zonestar_printer_extruder_3"
+        }
+    },
+
+    "overrides": {
+        "machine_name": { "default_value": "ZONESTAR Z9M4" },
+        "layer_height": { "default_value": 0.2 },
+        "machine_extruder_count": { "default_value": 4 },
+        "machine_heated_bed": { "default_value": true },
+        "machine_center_is_zero": { "default_value": false },
+        "machine_height": { "default_value": 400 },
```

Comment:
According to [this product page](https://www.xpita.com/product/zonestar-z9m4-newest-4-extruders-4-in-1-out-frame-corexy-struct-auto-level-3d-printer/), the height is 380mm, not 400mm. Which is correct?